tests/unit/widgets/BlockTest.php

```diff
@@ -28,13 +28,13 @@ public function testShouldTriggerInitEvent()
     {
         $initTriggered = false;
 
-        $block = new Block(
-            [
-                'on init' => function () use (&$initTriggered) {
-                    $initTriggered = true;
-                }
-            ]
-        );
+        $block = new Block($this->app);
+
+        //fixme: is this event ok ??
+        $block->on('init', function ($event) use (&$initTriggered) {
```

Comment:
Event name could be specified as `WidgetEvent::INIT`.

Reply:
It's not OK since INIT is now triggered in constructor so it's not possible to attach it after `new Block()`. @hiqsol any idea on how to deal with it?